src/components/cards/BldgCardImage.tsx

```diff
@@ -1,45 +0,0 @@
-import React from "react";
-
-// images this component may need to use -- is there a better way?
-import archeryHut from "../../assets/images/archeryHut-01.jpg";
-import healingChamber from "../../assets/images/healingChamber-01.jpg";
-import mealHall from "../../assets/images/mealHall-01.jpg";
-import swordsmithy from "../../assets/images/swordsmithy-01.jpg";
-import townCenter from "../../assets/images/townCenter-01.jpg";
-import { BuildingType } from "../../types";
-
-interface BldgCardImageProps {
-  buildingType: BuildingType;
-}
-
-/* FIXME: Devise a way to show symbol if no image path found */
-
-export default function BldgCardImage({ buildingType }: BldgCardImageProps) {
-  let src;
-  switch (buildingType) {
-    case "archeryHut":
-      src = archeryHut;
-      break;
-    case "healingChamber":
-      src = healingChamber;
-      break;
-    case "mealHall":
-      src = mealHall;
-      break;
-    case "swordsmithy":
-      src = swordsmithy;
-      break;
-    case "townCenter":
-      src = townCenter;
-      break;
-    default:
-      src = "";
-      break;
-  }
```

Comment:
Not a fan of this to be honest. Instead of having to pass in a `buildingType` why not pass in an `imagePath` instead? That way you don't have to do switch statements to determine the image path, you don't have to mess with types either. You literally just pass in `../assets/images/mealhall.jpg` and you are done.

Better yet, use this with the above idea of re-exporting them through `images.ts`.

Then you can just do `src={props.imagePath}`.